src/components/common/topnav/TopNav.css.ts

```diff
@@ -32,6 +36,7 @@ export const NavLink = styled(Nav.Link)`
 export const TopNavLogo = styled.span`
 	text-align: left;
 	flex-grow: 1;
+	z-index: 50;
```

Comment:
I'd asked about this before. I'm assuming this is where things panned out?

Reply:
yup, I have seen a couple approaches to using `z-index` and I'm not sure what the best practice is. Maybe we should use 1, 2, 3... etc. To be on the safe side I put it really high (50) so it would be on top of everything but I'll look around and see if there is anything else using it. Do you have an feelings one way or the other for that?

Reply:
I don't, so let it be.